sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/infer_geneformer.py

```diff
@@ -47,6 +47,7 @@ def infer_model(
     num_nodes: int = 1,
     num_dataset_workers: int = 0,
     config_class: Type[BioBertConfig] = GeneformerConfig,
+    bypass_tokenizer_vocab: bool = False,
 ) -> None:
     """Inference function (requires DDP and only training data that fits in memory)."""
     # This is just used to get the tokenizer :(
```

Comment:
the line right below this downloads data in the old format

Reply:
Yup, I believe it's just to generate the tokenizer so it shouldn't matter, but will go ahead and change it for consistency!